python/lsst/ts/MTAOS/mtaos.py

```diff
@@ -238,6 +246,33 @@ async def configure(self, config):
         if dof_state0 is not None:
             self.model.ofc_data.dof_state0 = dof_state0
 
+        if hasattr(config, "wep_config"):
+            with open(self.config_dir / config.wep_config) as fp:
+                self.wep_config = yaml.safe_load(fp)
+                try:
+                    tmp_visit_info = VisitInfo(
+                        boresightRaDec=SpherePoint(
+                            0.0 * degrees,
+                            0.0 * degrees,
+                        ),
+                        boresightRotAngle=0.0 * degrees,
+                    )
+                    wep_config_expanded = self.model.expand_wep_configuration(
+                        self.wep_config,
+                        tmp_visit_info,
+                    )
+                    self.model.wep_configuration_validation.validate(
+                        wep_config_expanded
+                    )
+                except Exception as e:
+                    self.log.exception("Failed to validate WEP configuration.")
+                    raise salobj.ExpectedError(
```

Comment:
Please add the `salobj.ExpectedError` in the doc string of this function. Thanks!